kornia/core/data/tensor_base.py

```diff
@@ -0,0 +1,25 @@
+from typing import List, Union
+from torch import Tensor
```

Comment:
Let's do `kornia.Tensor` and point to `torch.Tensor` in `kornia.core.stubs` (or similar). We eventually and optional use `ivy`. /cc @djl11 track this effort @ducha-aiki